src/views/Home/index.tsx

```diff
@@ -71,6 +78,29 @@ const HomeView = (): JSX.Element => {
     }
   }, [categories])
 
+  useEffect(() => {
+    handleGetPacks();
+  }, [])
+
+  const handleGetPacks = async () => {
+    const packs = await getPacks();
+
+    if (packs.data.success) {
+      setPackNames(packs.data.data.map<OptionType>((elem) => {
+        return { label: elem.name.substring('Polygon'.length), value: elem.identifier } as OptionType
```

Comment:
`'Polygon'.length`
You should probably declare this as a constant somewhere above with other constants and not use it here (also mind that it's iterated)
`const POLYGON_PREFIX_LENGTH = 7`